src/aliceVision/image/io_test.cpp

```diff
@@ -22,8 +22,23 @@ using namespace aliceVision;
 using namespace aliceVision::image;
 using std::string;
 
+#ifdef WIN32
+int setenv(const char* name, const char* value, int overwrite)
+{
+    int errcode = 0;
+    if (!overwrite) {
+        size_t envsize = 0;
+        errcode = getenv_s(&envsize, NULL, 0, name);
+        if (errcode || envsize) return errcode;
+    }
+    return _putenv_s(name, value);
+}
+#endif
+
+int err = setenv("ALICEVISION_ROOT", std::string(THIS_SOURCE_DIR).c_str(), 1);
```

Comment:
I think instead of overriding `ALICEVISION_ROOT` via setenv, we should retrieve the environmental variable via a function in which we could add overriding functionality. Something like:

```
static std::string aliceVisionRootOverride;

std::string getAliceVisionRoot()
{
    if (!aliceVisionRootOverride.empty())
        return aliceVisionRootOverride;
    const char* value = std::getenv("ALICEVISION_ROOT");
    return value ? value : "";
}

void setAliceVisionRootOverride(const std::string& value)
{
    aliceVisionRootOverride = value;
}
```

Reply:
I've just did a PR with this: https://github.com/alicevision/AliceVision/pull/1268